.github/workflows/test-promot.yaml

```
@@ -0,0 +1,34 @@
+---
+name: test-promot
```

change the file name and the workflow name
```suggestion
name: test-promote
```